client/task.go

```diff
@@ -241,10 +242,13 @@ func (t *task) Pid() uint32 {
 }
 
 func (t *task) Start(ctx context.Context) error {
-	ctx, span := tracing.StartSpan(ctx, "task.Start",
-		tracing.WithAttribute("task.id", t.ID()),
-	)
+	ctx, span := tracing.StartSpan(ctx, tracing.Name("client.task", "Start"),
+		tracing.WithAttribute("task.id", t.ID()))
 	defer span.End()
+	if ns, err := namespaces.NamespaceRequired(ctx); err == nil {
```

Comment:
I don't think we need this for CRI. CRI are using that fixed namespace - `k8s.io` - in containerd.